pom.xml

```diff
@@ -183,16 +183,9 @@
 				<optional>true</optional>
 			</dependency>
 			<dependency>
-				<groupId>servlets.com</groupId>
-				<artifactId>cos</artifactId>
-				<version>05Nov2002</version>
-				<scope>provided</scope>
-				<optional>true</optional>
-			</dependency>
-			<dependency>
-				<groupId>commons-fileupload</groupId>
-				<artifactId>commons-fileupload</artifactId>
-				<version>1.5</version>
+				<groupId>org.apache.commons</groupId>
+				<artifactId>commons-fileupload2-jakarta-servlet6</artifactId>
```

Comment:
Shouldn't we use `commons-fileupload2` here for Servlet 5 for now?

https://dist.apache.org/repos/dist/release/commons/fileupload/RELEASE-NOTES.txt

Reply:
I would rather go with commons-fileupload2-jakarta-servlet5 for now, since apparently someone has put a lot of care and attention to getting this right, and newer version will probably adhere to the scheme of artifact names established by these efforts.